proof-systems/Cargo.toml

```diff
@@ -50,7 +50,8 @@ algebra = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "ref
 r1cs-crypto = { path = "../r1cs/gadgets/crypto" }
 
 [features]
-print-trace = [ "bench-utils/print-trace" ]
+#TODO: with print-trace UTs fails
```

Comment:
There is some code containing traces that is executed in parallel, messing the whole thing up. We must disable it when testing ( I thought in the CI script this was already handled)

Reply:
Done